pkg/runtime/wfengine/wfengine_test.go

```diff
@@ -19,183 +19,31 @@ package wfengine_test
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"sort"
-	"sync"
 	"testing"
 	"time"
 
-	"github.com/google/uuid"
 	"github.com/microsoft/durabletask-go/api"
 	"github.com/microsoft/durabletask-go/backend"
 	"github.com/microsoft/durabletask-go/task"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
 
 	"github.com/dapr/components-contrib/state"
 	"github.com/dapr/dapr/pkg/actors"
 	"github.com/dapr/dapr/pkg/config"
 	"github.com/dapr/dapr/pkg/resiliency"
 	"github.com/dapr/dapr/pkg/runtime/wfengine"
+	daprt "github.com/dapr/dapr/pkg/testing"
 	"github.com/dapr/kit/logger"
 )
 
 const testAppID = "wf-app"
 
-// The fake state store code was copied from actors_test.go.
-// TODO: Find a way to share the code instead of copying it, if it makes sense to do so.
```

Comment:
Reviewers: this code was moved to the testing package to address this TODO and reduce code duplication